### ISSUE TITLE
slug is not correctly created with locale specific types

### ISSUE THREAD

**Issue by nordbit** [nitely/Spirit#54]
from:
```
Siedemnaście mgnień wiosny
```

to:
```
siedemnaście-mgnień-wiosny
```
should be:
```
siedemnascie-mgnien-wiosny
```

You can give a try 'unidecode' module
branch experimental

**Comment by nitely** [comment]
Unicode URLs are valid nowadays, isn't?

**Comment by nordbit** [comment]
Indeed. But is also highly questionable. Slug is for typing link for all users, my keyboard for example have not germans diacritic signs. If it is not a bug but feature, its ok. Great project BTW. I looking for forum module with integration in my existing project. So far Spirit is the best.

**Comment by nitely** [comment]
Yep, it's a feature, it was requested many time by Asian guys (I believe). I think most people copy&paste urls instead of typing them, but I guess it's possible someone may want to type it for some reason, in that case you are probably right.

> Great project BTW. I looking for forum module with integration in my existing project. So far Spirit is the best.

Thanks :smile: it will get way better!

**Comment by henri-hulski** [comment]
I like Spirit's design very much and I am excited about the new features. 
Last night they released Mezzanine 0.4, which supports Django 1.8.
So I'm looking forward to upgrade my project to Django 1.8, Mezzanine 0.4 and Spirit 0.3.

Is there much work left before the release? 

Thanks a lot for your awesome forum, 
Regards, Henri
